tests/configfiles/test_files/default_rule.yaml

```diff
@@ -0,0 +1,225 @@
+            - "_clear_packaging_caches_and_temporary_package_downloads" 
+        -
+          label: "no_apt-get_clean"
+          regex: /apt-get install(?!.+clean)/g
+          level: "info"
+          message: "apt-get clean is not used"
+          description: "the apt-get cache will remain in this layer making the layer unnecessarily large"
+          reference_url: 
+            - "http://docs.projectatomic.io/container-best-practices/#"
+            - "_clear_packaging_caches_and_temporary_package_downloads" 
+        -
+          label: "privileged_run_container"
+          regex: /privileged/
+          level: "warn"
+          message: "a privileged run container is allowed access to host devices"
+          description: "Does this run need to be privileged?"
+          reference_url: 
+            - "http://docs.docker.com/engine/reference/run/#"
+            - "runtime-privilege-and-linux-capabilities"
+        -
+          label: "installing_ssh"
+          regex: /openssh-server/
+          level: "warn"
+          message: "installing SSH in a container is not recommended"
+          description: "Do you really need SSH in this image?"
+          reference_url: "https://github.com/jpetazzo/nsenter"  
+        - 
+          label: "no_ampersand_usage"
+          regex: / ; /
+          level: "info"
+          message: "using ; instead of &&"
+          description: "RUN do_1 && do_2: The ampersands change the resulting evaluation into do_1 and then do_2 only if do_1 was successful."
+          reference_url:
+            - "http://docs.projectatomic.io/container-best-practices/#"
+            - "#_using_semi_colons_vs_double_ampersands"
+    EXPOSE: 
+      paramSyntaxRegex: /^[\d-\s\w/\\]+$/
+      rules: []
+    ENV: 
+      paramSyntaxRegex: /^[\w-$/\\=\"[\]{}@:,'`\t. ]+$/
+      rules: []
+    ADD: 
+      paramSyntaxRegex: /^~?([\w-.~:/?#\[\]\\\/*@!$&'()*+,;=.{}"]+[\s]*)+$/
+    COPY: 
+      paramSyntaxRegex: /.+/
+      rules: []
+    ENTRYPOINT:
+      paramSyntaxRegex: /.+/
+      rules: []
+    VOLUME:
+      paramSyntaxRegex: /.+/
+      rules: []
+    USER: 
+      paramSyntaxRegex: /^[a-z0-9_][a-z0-9_-]{0,40}$/
+      rules: []
+    WORKDIR: 
+      paramSyntaxRegex: /^~?[\w\d-\/.{}$\/:]+[\s]*$/
+      rules: []
+    ONBUILD: 
+      paramSyntaxRegex: /.+/
+      rules: []
+  required_instructions: 
+    - 
+      instruction: "EXPOSE"
+      count: 1
+      level: "info"
+      message: "There is no 'EXPOSE' instruction"
+      description: "Without exposed ports how will the service of the container be accessed?"
+      reference_url: 
+        - "https://docs.docker.com/engine/reference/builder/"
+        - "#expose"
+    - 
+      instruction: "CMD"
+      count: 1
+      level: "info"
+      message: "There is no 'CMD' instruction"
+      description: "None"
+      reference_url: 
+        - "https://docs.docker.com/engine/reference/builder/"
+        - "#cmd"
```

Comment:
wheee that's a large config. Did you copy it from somewhere? In that case we maybe have to write somewhere that this is from source "xyz", depending on the license.
Anyway, can we maybe shorten it? I guess you don't exploit all those settings here in the tests :) But not necessary to shorten it^^

Reply:
Yeah, this is the one for which `dockefile_lint` checks by default. So, I guess this should not be shortened. I took it from the linter's repo itself which includes some sample rule files. It's MIT licensed. Where should I add a reference to it?

Reply:
good questions... I think MIT is compatible in this regard here with GPL3, but can you google around how to include MIT code into GPL3?

Reply:
All I can find is:
- Any code licensed under MIT can be redistributed as a part of another project under any license.
- The license text & copyright notice should be included with any unmodified MIT licensed code.
So, where do I need to add the same exact copy of the MIT license that dockerfile_lint has?